ui/app/src/components/Header/SearchBar.tsx

```diff
@@ -0,0 +1,223 @@
+    } else {
+      return [];
+    }
+  }, [kvSearch, props.list, props.query]);
+
+  return filteredList.length === 0 ? (
+    <></>
+  ) : (
+    <Box sx={{ display: 'flex', flexDirection: 'column' }}>
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'row',
+          alignItems: 'center',
+          justifyContent: 'flex-start',
+          marginBottom: 1,
+          marginLeft: 0.5,
+        }}
+      >
+        <ViewDashboardIcon sx={{ marginRight: 0.5 }} fontSize="medium" />
+        <Typography variant="h3">Dashboards</Typography>
+      </Box>
+      {filteredList.slice(0, sizeList).map((search) => (
+        <Button
+          variant="outlined"
+          sx={{
+            display: 'flex',
+            justifyContent: 'space-between',
+            marginBottom: 1,
+            marginLeft: 1,
+            marginRight: 1,
+          }}
+          component={RouterLink}
+          onClick={props.onClick}
+          to={`/projects/${search.original.metadata.project}/dashboards/${search.original.metadata.name}`}
+          key={`dashboard-${search.original.metadata.project}-${search.original.metadata.name}`}
+        >
+          <Box sx={{ display: 'flex' }} flexDirection="row" alignItems="center">
+            <span
+              dangerouslySetInnerHTML={{
+                __html: kvSearch.render(search.original, search.matched, {
+                  pre: '<strong style="color:darkorange">',
+                  post: '</strong>',
+                  escapeHTML: true,
+                }).metadata.name,
+              }}
+            />
+          </Box>
+          <Chip label={`${search.original.metadata.project}`} size="small" variant="outlined" />
+        </Button>
+      ))}
+    </Box>
+  );
+}
+
+export function SearchBar() {
+  const isMobileSize = useIsMobileSize();
+  const [open, setOpen] = React.useState(false);
+  const [query, setQuery] = React.useState('');
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const dashboardQueryResult = useDashboardList(undefined, true);
+  const projectsQueryResult = useProjectList();
+  return (
+    <Paper sx={{ flexGrow: 2, maxWidth: isMobileSize ? '50%' : '30%' }}>
+      <Button size="small" fullWidth sx={{ display: 'flex', justifyContent: 'space-between' }} onClick={handleOpen}>
+        <Box sx={{ display: 'flex' }} flexDirection="row" alignItems="center">
+          <Magnify sx={{ marginRight: 0.5 }} fontSize="medium" />
+          <Typography>Search ...</Typography>
+        </Box>
+        <Chip label={`${shortcutCTRL()}+k`} size="small" />
+      </Button>
+      <Modal
+        open={open}
+        onClose={handleClose}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+        style={{ display: 'flex', justifyContent: 'center' }}
+      >
+        <Paper
+          elevation={0}
+          sx={{
+            height: '70%',
```

Comment:
On mobile, we should use 90%

Reply:
I think on mobile 70% is still good. Otherwise it becomes painful to go out of the modal :(